qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/internal/java/AbstractQuteTemplateLinkCollector.java

```diff
@@ -208,8 +216,9 @@ public boolean visit(TypeDeclaration node) {
 	@Override
 	public boolean visit(RecordDeclaration node) {
 		if (isImplementTemplateInstance(node)) {
+			TemplateNameStrategy templateNameStrategy = TemplateNameStrategy.ELEMENT_NAME;

```

Comment:
this can be inlined directly in the collectTemplateLink call

Reply:
fixed